### PR TITLE
Add AI PL homepage and skinny banners

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adding banners for the launch of AI 101 for Teachers:

**On code.org:**
- Homepage hero banner 
- Skinny banner on Teach page (https://code.org/teach)
- Both use the DCDO flag `ai-pl-launch-banners`

**On studio.code.org:**
- Announcements banner on Teacher logged in homepage (http://studio.code.org/home)

**Jira ticket:** [ACQ-698](https://codedotorg.atlassian.net/browse/ACQ-698) 

----

### Homepage hero banner (https://code.org)
Desktop

### Skinny banner on Teach page (https://code.org/teach)
Desktop (1)

### Announcements banner on Teacher logged in homepage (http://studio.code.org/home)
Teacher logged in homepage

### CODE REVIEW

File: pegasus/sites.v3/code.org/homepage.json [hunk 1]
```diff
@@ -116,6 +117,30 @@
           "img_src": "/images/banners/banner-curriculum-catalog-illustration-black.png"
         }
       ]
+    },
+    "ai-pl-launch-2023": {
+      "dcdo": "ai-pl-launch-banners",
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/homepage.json:122]: Does setting this dcdo variable here put it behind that flag?

**Review by TurnerRiley — APPROVED** [review]
This looks amazing! Nice job keeping track of all the moving parts! I just had one question more for my own understanding

File: pegasus/sites.v3/code.org/homepage.json (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/homepage.json:122]: Yep!

Comment (TurnerRiley) [reply pegasus/sites.v3/code.org/homepage.json:122]: Cool, TIL!

**Pull request merged**